sourcegraph/sourcegraph
# Implement browser extension Sourcegraph URL dropdown for ease of URL switching

Closes https://github.com/sourcegraph/sourcegraph/issues/29030.
Closes https://github.com/sourcegraph/sourcegraph/issues/25778.

#### Description

This PR:
- Implement browser extension Sourcegraph URL dropdown for ease of URL switching
- Updates options page UI

#### How to test

1. `sg run bext`
2. Open browser extension options page
3. Check that the input field suggests **all previously successfully validated URLs**

#### Screenshots
| Before | After |
| -: | :- |
|![image](https://user-images.githubusercontent.com/6717049/148399252-95d2de29-af50-41ca-8bbf-f3bdeec4ed95.png) | ![image](https://user-images.githubusercontent.com/6717049/148399303-cc6b53c6-6d5b-4077-befa-54d3185e5d3b.png) |
| n/a | ![image](https://user-images.githubusercontent.com/6717049/148399399-09b3c0db-55ef-4a1a-b805-dc7e586e601d.png) |
| ![image](https://user-images.githubusercontent.com/6717049/148399557-630a3a6c-4fcf-4c21-92b7-6599258bb3d3.png) | ![image](https://user-images.githubusercontent.com/6717049/148399524-fd9116a0-c6ad-4cc2-b227-d48720f41618.png) |
|![image](https://user-images.githubusercontent.com/6717049/148399603-a74c22cb-2804-4ad9-a44a-3e20bf9000ab.png) | ![image](https://user-images.githubusercontent.com/6717049/148399632-5776f877-c8e0-4053-a0cc-fe5b16b92ff8.png) |

#### Before merging
